contributing/adr/0005-use-erb-for-component-templates.md

```diff
@@ -0,0 +1,32 @@
+# 5. Use ERB for component templates
+
+Date: 2022-07-26
+
+## Status
+
+Proposed
+
+## Context
+
+We've seen output buffer issues when using out components outside of Rails within our Bridgetown docs. This happens where the components using a Haml template and where there is any kind of nested content block (e.g. tag helpers or composition). See https://github.com/citizensadvice/design-system/pull/2207 for an example of the issue and https://github.com/citizensadvice/design-system/pull/2207 demonstrating that using ERB for the template resolves any issues.
+
+It's possible to work around some of the issues by adjusting the Haml templates with delicate use of `capture_haml` or by adjusting other parts of the templates on a case by case basis but this feels fraught.
```

Comment:
It would be nice to expand on this slightly - not War and Peace but just what the pitfalls of it are / why it feels `fraught`.